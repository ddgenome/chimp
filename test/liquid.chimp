## liquid-phase only mechanism
mechanism "liquid.mech"
## parameter input
parameter "liquid.par"
## liquid-phase task
task "liquid.task"
