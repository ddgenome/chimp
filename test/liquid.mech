# a liquid-phase mechanism
A + B -> k_arrhenius(A1f, E1f) <- k_arrhenius(A1r, E1r) C;
C -> k_arrhenius(A2f, E2f) <- k_arrhenius(A2r, E2r) D;
C + D -> k_arrhenius(A3f, E3f) <- k_arrhenius(A3r, E3r) E;
