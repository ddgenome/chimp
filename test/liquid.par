# parameter input file for gas-phase mechanism
A1f	1.5e1	# m^3/mol/s
E1f	4.5e4	# J/mol
A1r	3.0e3	# 1/s
E1r	4.9e4	# J/mol
A2f	5.0e2	# 1/s
E2f	3.4e4	# J/mol
A2r	4.0e1	# 1/s
E2r	3.8e4	# J/mol
A3f	5.0e0	# m^3/mol/s
E3f	4.2e4	# J/mol
A3r	7.0e1	# 1/s
E3r	4.9e4	# J/mol
