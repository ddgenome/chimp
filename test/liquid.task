# -*- text -*-
# liquid-phase task input
begin model liquid
  output "liquid.out"
  begin integrator kmc
    scale 1.0e20
    begin state
      begin quantity
	c[A] = 1.0e3		# mol/m^3
	c[B] = 1.0e3		# mol/m^3
      end quantity
      begin output
	(1.0e2 5.0e2 1.0e2)
      end output
      begin reactor batch
	temperature 3.0e2	# K
	pressure 1.0e0		# Pa
	volume 1.0e0		# m^3
	rate_numerator moles
	rate_denominator volume
	fluid_quantity concentration
      end reactor
    end state
  end integrator
end model
