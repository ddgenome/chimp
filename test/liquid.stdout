chimp 0.1
Copyright (C) 2004 David Dooling <banjo@users.sourceforge.net>

There is ABSOLUTELY NO WARRANTY; see file COPYING for details.
This is free software, and you are welcome to redistribute it
under certain conditions; see file COPYING for details.

Report bugs to http://sourceforge.net/projects/chimp/.
chimp: performing task liquid.par... completed; 0u 0s 0w
chimp: performing task liquid... completed; 6.13u 0s 6.13w
